htlcswitch/switch.go

```diff
@@ -470,7 +470,28 @@ func (s *Switch) SendHTLC(firstHop lnwire.ShortChannelID, paymentID uint64,
 	// Send packet to link.
 	packet.circuit = circuit
 
-	return s.route(packet)
+	// User have created the htlc update therefore we should find the
+	// appropriate channel link and send the payment over this link.
+	link, linkErr := s.handleLocalAddHTLC(packet, htlc)
```

Comment:
Pre-existing, but this method name really doesn't match the operation. It's more like `localEligibleLink` or something like that.

Reply:
Renamed to `getLocalLink`